.rubocop.yml

```diff
@@ -281,3 +281,8 @@ Style/TrailingCommaInLiteral:
 
 Style/SingleLineBlockParams:
   Enabled: false
+
+Layout/IndentHash:
+  Enabled: true
+  Exclude:
+    - '**/*.html.slim*'
```

Comment:
We have rubocop enabled for slim-lint. It runs rubocop on the files after slim processes them. I added a multiline hash to one and since slim removes all indentation this cop got angry.

There is a [`ignored_cops` options for slim-lint](https://github.com/sds/slim-lint/blob/master/lib/slim_lint/linter/README.md#rubocop), but adding one cop to that appears to re-enable all of the cops that slim-lint ignores by default.